spec/test_files/get_commit_sha_0e67cdb1c7582ce3e2fd00367acc24db6242c6d6.json

```diff
@@ -33,7 +33,17 @@
                 "type": "User",
                 "site_admin": false
             },
-            "labels": [],
+            "labels": [
```

Comment:
Maybe have one test where `labels` is empty? Since for other SDKs that use this but don't put labels that will be the common case.